client/components/errorLabel/errorLabel.css

```diff
@@ -1,17 +1,15 @@
-.button-error {
+@import '../typography/typography.css';
+
+.error-message {
     display: flex;
+    justify-content: center;
     flex-direction: row;
     align-items: center;
     padding: 4px 8px;
     width: 49px;
     height: 24px;
-    background: #FFF2F6;
+    background: var(--invalid-background);
     border-radius: 8px;
-    font-family: 'IBM Plex Sans';
-    font-style: normal;
-    font-weight: 600;
-    font-size: 14px;
-    color: #B30038;
     border: none;
     }
```

Comment:
this changes are also not related to button link can you remove this from PR?

Reply:
Good afternoon, good.